benchmarks/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
             project(':grpc-stub'),
             project(':grpc-protobuf'),
             project(':grpc-testing'),
+            project(':grpc-xds'),
```

Comment:
Did you test with xDS? It uses ALTS credentials and grpc-alts will use grpc-netty-shaded. So probably we need
```
project(path: ':grpc-xds', configuration: 'shadow')
```
as in grpc-interop-testing project.

Reply:
xds works fine (well enough to get an error from TD). Alts would trigger using netty-shaded for the channel. You'd only see a problem if you tried to use XdsServerBuilder with AltsServerCredentials, which both have their ProtocolNegotiators.

But you are right, we want to use the artifact everyone else uses.